### ISSUE TITLE
CRLF in code files: Difference between deno run and deno compile

### ISSUE THREAD

**Issue by fknipp** [denoland/deno#10282]
## Test case

Just take this input file (_crlf-test.ts_) and save it with CRLF line endings:

```typescript
const content1 = `first line\r\nsecond line\r\nthird line`;

const content2 = `first line
second line
third line`;

console.log({ content1, content2 });
```

Running it with ` deno run crlf-test.ts` returns

```
{
  content1: "first line\r\nsecond line\r\nthird line",
  content2: "first line\nsecond line\nthird line"
}
```

Compiling it using `deno compile --unstable crlf-test.ts` and running the executable returns

```
{
  content1: "first line\r\nsecond line\r\nthird line",
  content2: "first line\r\nsecond line\r\nthird line"
}
```

## Observation

Line endings are treated differently between running and compiling.

## Temporary solution

I could solve the issue in my project by changing all line endings to LF.

## Used environment

The issue can be observed on Windows as well as on Linux.

### Windows

```
deno 1.9.0 (release, x86_64-pc-windows-msvc)
v8 9.1.269.5
typescript 4.2.2
```

### Linux

```
deno 1.9.0 (release, x86_64-unknown-linux-gnu)
v8 9.1.269.5
typescript 4.2.2
```

Thanks for providing Deno 😃

**Comment by Liamolucko** [comment]
cc @kdy1

**Comment by kdy1** [comment]
Fixed with new version of `swc_ecma_codegen`.

**Comment by lucacasonato** [comment]
Should be resolved in 1.11 which will be released later today.
